cluster-autoscaler/cloudprovider/aws/aws_manager.go

```diff
@@ -458,6 +469,26 @@ func extractAllocatableResourcesFromAsg(tags []*autoscaling.TagDescription) map[
 	return result
 }
 
+func extractAllocatableResourcesFromTags(tags map[string]string) map[string]*resource.Quantity {
+	result := make(map[string]*resource.Quantity)
+
+	for k, v := range tags {
+		splits := strings.Split(k, "k8s.io/cluster-autoscaler/node-template/resources/")
+		if len(splits) > 1 {
+			label := splits[1]
+			if label != "" {
+				quantity, err := resource.ParseQuantity(v)
+				if err != nil {
+					continue
```

Comment:
Did we want to log something here about an unparseable quantity string? I would think this might be something difficult to diagnose if there is no signal to the end user anywhere that their tag is malformed :)